t/app/controller/admin.t

```diff
@@ -25,7 +25,7 @@ my $dt = DateTime->new(
     second => 23
 );
 
-my $report = FixMyStreet::App->model('DB::Problem')->find_or_create(
+my $report = FixMyStreet::DB->resultset('Problem')->find_or_create(
```

Comment:
I'm not clear why this change is happening.

Reply:
Shouldn't really be using the full App in a test if all we need is the DB schema, for performance, but in particular, it's because an upgraded package somewhere now throws a warning if you use App's find_or_create but use a User created via DB->resultset, saying that seems a bit odd.